### DISCUSSION TITLE
不使用点击上传图片按钮，wangEditor如何在VUE2实现复制（截图）直接粘贴图片到编辑栏？

### DISCUSSION THREAD

**Discussion by gushenyi123** [wangeditor-team/wangEditor#4408] [Q&A]
不使用点击上传图片按钮，wangEditor如何在VUE2实现复制（截图）直接粘贴图片到编辑栏？

**Comment by wangfupeng1988** [comment]
配置好服务端地址，或者 `base64LimitSize` ，就可以粘贴截图。[官网 demo](https://www.wangeditor.com/demo/index.html) 就可粘贴截图。

文档： https://www.wangeditor.com/v5/menu-config.html#%E4%B8%8A%E4%BC%A0%E5%9B%BE%E7%89%87
